tests/test_basic.py

```diff
@@ -110,6 +111,112 @@ def test_histplot_density():
     return fig
 
 
+@pytest.mark.mpl_image_compare(style="default")
+def test_histplot_flow():
+    np.random.seed(0)
+    h = hist.Hist(hist.axis.Regular(20, 5, 15, name="x"), hist.storage.Weight())
+    h.fill(np.random.normal(10, 3, 400))
+    fig, axs = plt.subplots(1, 3, sharey=True, figsize=(15, 5))
+    axs = axs.flatten()
+
+    hep.histplot(h, ax=axs[0], flow="hint")
+    hep.histplot(h, ax=axs[1], flow="show")
+    hep.histplot(h, ax=axs[2], flow="sum")
+
+    axs[0].set_title("Default(hint)", fontsize=18)
+    axs[1].set_title("Show", fontsize=18)
+    axs[2].set_title("Sum", fontsize=18)
+    return fig
+
+
+@pytest.mark.mpl_image_compare(style="default")
+def test_histplot_hist_flow():
+    np.random.seed(0)
+    entries = np.random.normal(10, 3, 400)
+    h = hist.Hist(hist.axis.Regular(20, 5, 15, name="x"), hist.storage.Weight())
+    h2 = hist.Hist(hist.axis.Regular(20, 5, 15, name="x"), hist.storage.Weight())
+    h3 = hist.Hist(hist.axis.Regular(20, 5, 15, name="x"), hist.storage.Weight())
+    h4 = hist.Hist(hist.axis.Regular(20, 5, 15, name="x"), hist.storage.Weight())
+
+    h.fill(entries)
+    h2.fill(entries[entries < 15])
+    h3.fill(entries[entries > 5])
+    h4.fill(entries[(entries > 5) & (entries < 15)])
+    import uproot
+
+    f = uproot.recreate("flow_th1.root")
+    f["h"] = h
+    f["h2"] = h2
+    f["h3"] = h3
+    f["h4"] = h4
+    fig, axs = plt.subplots(2, 2, sharey=True, figsize=(10, 10))
+    axs = axs.flatten()
+
+    hep.histplot(h, ax=axs[0], flow="show")
```

Comment:
This doesn't need to get created when testing `hist`. Also instead of the bins existing and you not filling them, can you replace it with 
```
h = hist.Hist(hist.axis.Regular(20, 5, 15, name="x", flow=True), hist.storage.Weight())
h2 = hist.Hist(hist.axis.Regular(20, 5, 15, name="x", underflow=True, overflow=False), hist.storage.Weight())
h3 = hist.Hist(hist.axis.Regular(20, 5, 15, name="x", underflow=False, overflow=True), hist.storage.Weight())
h4 = hist.Hist(hist.axis.Regular(20, 5, 15, name="x", flow=False), hist.storage.Weight())
```

These can also iirc be shortened with `hist.new.Reg(20, 5, 15, name="x", underflow=True, overflow=False).Weight()`

Reply:
Hmm... the suggestion works. but if we convert the hist functions to ROOT TH1 the functions not act correctly...
The bins and value would mismatch for one-sided case

Reply:
Yeah, this won't work for TH1 since ROOT cannot properly store hists without flow bins

Reply:
so should I keep the previous structure or moving to the `hist` suggestion here?